app/src/forms/attestations/middleware/searchParameters.js

```diff
@@ -1,5 +1,4 @@
 const Problem = require('api-problem');
-const Models = require('../models');
```

Comment:
Think some of the fxns in this file are more util than middleware-specific. Like checking if a string in the API probably not specific to the /attestations/ folder

Reply:
Yup, but until they are needed elsewhere, I'd prefer to keep them where they are used.  Probably more stuff will get refactored out of attestations, but let's deal with it then.